doc/stories.rst

```diff
@@ -135,6 +135,37 @@ for my needs:
 Build System
 ------------
 
+`BMW <https://www.bmw.com/de/index.html>`_ (Automotive)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+by `Mike Pagel <https://github.com/moltob>`_ (2019-02-06)
+
+We are responsible for the development of the next generation instrument
+cluster software at BMW. While we use CMake for the actual build of libraries
+and applications, we have learned in the past that CMake *can* do almost
+anything, but it maybe *shouldn’t*.
+
+`CMake` is optimized for all tasks around compiler toolchain control, but the
```

Comment:
reST expects double backticks for inline code.

    ``CMake`` is optimized for…